test/puppetlabs/trapperkeeper/services/authorization/authorization_core_test.clj

```diff
@@ -69,6 +69,25 @@
    :match-request {:path "/path" :type "path"}})
 
 (deftest invalid-configs-fail
+  (testing "Configuration section required"
+    (is (thrown-with-msg?
+         IllegalArgumentException
+         #"Missing authorization service configuration."
+         (validate-auth-config! nil))))
+
+  (testing "Version number required in configuration"
+    (is (thrown-with-msg?
+         IllegalArgumentException
+         #"Unsupported or missing version in configuration file.*"
+         (validate-auth-config! {:rules []}))))
```

Comment:
I'm going to add two more iterations of this test against a non-zero number and a non-number value